ipaserver/install/server/__init__.py

```diff
@@ -548,6 +560,10 @@ def dm_password(self, value):
     def admin_password(self, value):
         validate_admin_password(value)
 
+    # always run sidgen task and do not allow adding agents on first master
+    add_sids = True
```

Comment:
The problem is that (for some reason) the 389-ds is populated by several default groups during bootstrap. One of these groups is `cn=editors` which is POSIX one has no SID assigned when I hardcoded `add_sids` to False (`cn=admins` have SID assinged directly by CIFS installer since it has to have special RID of 512).

@rcritten is there any reason why are these groups created during bootstrap and not during LDAP update phase?